src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -191,7 +215,14 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
   const renderedTables = useMemo(() => {
     if (isSeatedAtTable) return;
 
-    return tables.map((table: Table, index: number) => (
+    // @debt refactor table properties into one place
+    const tablesToShow = showAvailableTables
+      ? tables.filter(
+          (table) => !(isFullTable(table) || isLockedTable(table.title))
```

Comment:
`isLockedTable(table.title)` should be using `table.reference`. I'm pretty sure this won't work if you edit the table and change the title.

Reply:
fixed in 5107070527cdd346077ab84f3249ad327e1ab5d7